reference-implementation/lib/writable-stream.js

```diff
@@ -747,6 +747,21 @@ function WritableStreamDefaultControllerClose(controller) {
   WritableStreamDefaultControllerAdvanceQueueIfNeeded(controller);
 }
 
+function WritableStreamDefaultControllerGetChunkSize(controller, chunk) {
```

Comment:
This appears to be called from only one place, so inlining it fits better with our style.